user_sync/app.py

```diff
@@ -72,9 +72,18 @@ def process_args():
     parser.add_argument('--generate-remove-list',
                         help='processing similar to --remove-nonexistent-users except that rather than performing removals, a file is generated (with the given pathname) listing users who would be removed. This file can then be given in the --remove-list argument in a subsequent run.',
                         metavar='output_path', dest='remove_list_output_path')
-    parser.add_argument('-d', '--remove-list',
-                        help='specifies the file containing the list of users to be removed. Users on this list are removeFromOrg\'ed on the Adobe side.',
+    parser.add_argument('--remove-list',
+                        help='specifies the file containing the list of users to be removed. Users on this list are removed from the organization on the Adobe side.',
                         metavar='input_path', dest='remove_list_input_path')
+    parser.add_argument('--delete-nonexistent-users',
+                        help='Causes the user sync tool to delete users that exist on the Adobe side if they are not in the customer side AD. If the account is an Adobe account, it is remove from org\'d, but not deleted.',
```

Comment:
users -> user accounts,
Adobe account -> Adobe ID account
remove from org'd -> removed from the Adobe-side organization